src/main/controllers/MediaAccountRejectionReasonsController.ts

```diff
@@ -0,0 +1,50 @@
+import { PipRequest } from '../models/request/PipRequest';
+import { Response } from 'express';
+import { cloneDeep } from 'lodash';
+import path from 'path';
+import { MediaAccountApplicationService } from '../service/mediaAccountApplicationService';
+
+const mediaAccountApplicationService = new MediaAccountApplicationService();
+const url = 'media-account-rejection-reasons';
+
+export default class MediaAccountRejectionReasonsController {
+    public async get(req: PipRequest, res: Response): Promise<void> {
+        if (req.query.applicantId) {
+            const applicantId = req.query.applicantId;
+
+            const fs = require('fs');
+            const rejectReasons = JSON.parse(
```

Comment:
Rather than needing to read this in or parse, you can import the JSON file directly at the top of the class using require('../resources/<FileName.json>')

Reply:
beautiful. Much nicer solution!

Reply:
I've also done this to the stuff in njkfilters. Can't believe I didn't know this! We can do it with all the controllers!